api/ingest.py

```diff
@@ -162,6 +162,12 @@ def get_manifests_from_submission(self, submission_uuid):
         entity_url = f'{self.url}/bundleManifests/search/findByEnvelopeUuid?uuid={submission_uuid}'
         return self._get_all(entity_url, 'bundleManifests')
 
+    def set_submission_status_archived(self, submission_uuid):
+        submission_json = self.get_submission_by_uuid(submission_uuid)
+        if submission_json:
+            commit_archived_link = submission_json.get("_links", {}).get("commitArchived", {}).get("href", "")
```

Comment:
I think this is
```
commit_archived_link = submission_json.get("_links", {}).get("archived", {}).get("href", "")

``` 
Which will equal: `/submissionEnvelope/{id}/archivedEvent`

If you see here in core, the endpoint for requesting the state tracker to move the submission to archived state:
https://github.com/ebi-ait/ingest-core/blob/d0f250292d8dd7e4d214ceb330ed83cef6fbe4b3/src/main/java/org/humancellatlas/ingest/submission/web/SubmissionEnvelopeResourceProcessor.java#L192

The commit endpoint are used by state tracker if the request is accepted.

Ideally we should stop publishing the commit endpoints because I think they confuse the user.

Reply:
I pushed a change to fix this issue.